src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/index.html.twig

```diff
@@ -4,6 +4,14 @@
 {% block head %}
     {{ parent() }}
 
+    {{
+        add_css (
```

Comment:
Are we going to do a refactor later? and add all these to the layout.js file?

Reply:
I don't think it's worth the trouble using webpack for these applications. There is nothing to gain.